src/components/Icon/index.js

```diff
@@ -0,0 +1,56 @@
+import _ from 'underscore';
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import themeColors from '../../styles/themes/default';
+import variables from '../../styles/variables';
+import * as BrandAssets from './BrandAssets';
+import * as Expensicons from './Expensicons';
+
+const ICONS = _.extend(BrandAssets, Expensicons);
+
+const propTypes = {
+    /* The asset to render. */
```

Comment:
@marcaaron shouldn't all these comments start with `/**`, or if that hasn't been implemented yet, let's be consistent in the codebase at least and switch these to `//`.

Reply:
Yes! Ideally they will be `/**` but we can stick with `//` for now. I created an issue to convert them all, but it fell down the list of priorities. Can ask a contributor to help with this next week. https://github.com/Expensify/Expensify/issues/151907

Reply:
Cool, I'll just switch it back to `//` for now.